tests/test_interface.rb

```diff
@@ -958,6 +959,44 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
```

Comment:
Instead of using the `config` api, please use the `Interface` and `VxlanGlobal` providers to accomplish the same task.